### PR TITLE
Fixed the people picker dismiss problem

### PR DESCRIPTION
**Pull request by joschect**
The people picker was dismissing because onBlurCapture was getting events that didn't have a related target since the current target was window. This fix mirrors the way that blur/lostfocus is handled by other components.

### CODE REVIEW

**Comment by dzearing** [comment]
In IE only, does relatedTarget not get populated sometimes?

**Comment by joschect** [comment]
relatedTarget is never populated in IE (and maybe firefox) for the onBlur event. This was changed in ie Edge apparently. https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/101237/ .

**Comment by joschect** [comment]
Apparently my link is bad, here is another one. https://connect.microsoft.com/IE/feedback/details/814285/focus-and-blur-events-should-have-relatedtarget

File: src/components/PeoplePicker/PeoplePicker.tsx [hunk 1]
```diff
@@ -176,7 +176,8 @@ export class PeoplePicker extends React.Component<IPeoplePickerProps, IPeoplePic
   private _onBlurCapture(ev) {
     // onBlur, relatedTarget refers to the element that got focus
     if (!this.refs.searchField.contains(ev.relatedTarget)
-      && !this.refs.pickerResults.contains(ev.relatedTarget) ) {
+      && !this.refs.pickerResults.contains(ev.relatedTarget)
+      && ev.currentTarget !== window ) {
       this._dismissPeoplePicker();
```
Comment (dzearing) [inline src/components/PeoplePicker/PeoplePicker.tsx]: lets flip this and check for losing focus to something not in the picker in order to dismiss.

File: src/utilities/Dom/DomUtils.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export function elementContains(parent: HTMLElement, child: HTMLElement) : boolean {
```
Comment (dzearing) [inline src/utilities/Dom/DomUtils.ts]: Could you just name this file dom.ts under /utilities?

File: src/components/PeoplePicker/PeoplePicker.tsx [hunk 3]
```diff
@@ -75,7 +76,7 @@ export class PeoplePicker extends React.Component<IPeoplePickerProps, IPeoplePic
   }
 
   public componentDidMount() {
-    this._events.on(window, 'blur', this._onBlurCapture, true);
```
Comment (dzearing) [inline src/components/PeoplePicker/PeoplePicker.tsx]: one of the things i was just chatting with Thomas about... these global events should really only be done when needed. Just mentioning.

**Comment by dzearing** [comment]
Approved

**Pull request merged**